ReactiveObjC/RACSignal.m

```diff
@@ -25,8 +25,56 @@
 #import "RACTuple.h"
 #import <stdatomic.h>
 
+#if DEBUG
+
+#import "RACSourceSymbolExtractor.h"
+
+@interface RACSignal ()
+
+@property (readonly, nonatomic) NSArray<NSString *> *initializationCallStackSymbols;
```

Comment:
These should exist only in `DEBUG`

Reply:
it already does - changed it to be more explicit